prow/cmd/deck/jobs.go

```diff
@@ -189,7 +189,7 @@ func (ja *JobAgent) update() error {
 			ft: j.Status.CompletionTime,
 		}
 		if !nj.ft.IsZero() {
-			nj.Finished = nj.ft.Format("15:04:05")
+			nj.Finished = nj.ft.Format(time.Stamp)
```

Comment:
Actually, while `time.Stamp` mimics the `started` field, `time.RFC3339Nano` seems provides the most detailed information for programmatic use and I don't believe this is used anywhere else anyhow. I will change this.